### PR TITLE
docs: update translation on notification page for zh-CN

### PR DESCRIPTION
**Pull request by xgitea**
docs: update translation on notification page for zh-CN

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM cc @lunny

**Comment by lunny** [comment]
But you have to add English for `notification` as default.

File: options/locale/locale_zh-CN.ini [hunk 1]
```diff
@@ -13,6 +13,7 @@ version=当前版本
 page=页面
 template=模板
 language=语言选项
+notifications = 通知
```
Comment (appleboy) [inline options/locale/locale_zh-CN.ini:16]: Missing English key.

**Review by appleboy — CHANGES_REQUESTED** [review]

File: options/locale/locale_zh-CN.ini (hunk 1 above)
Comment (xgitea) [reply options/locale/locale_zh-CN.ini:16]: hi,@appleboy @lunny this english key exist.
[https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini](https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini)

page = Page
template = Template
language = Language
notifications = Notifications
create_new = Create...

Comment (appleboy) [reply options/locale/locale_zh-CN.ini:16]: OK

**Review by appleboy — APPROVED** [review]

**Comment by tboerger** [comment]
Hard to review or trust for me, that's up to @lunny and @appleboy

**Comment by lunny** [comment]
LGTM

**Comment by lunny** [comment]
Let L-G-T-M work

**Pull request merged**